runtime/oti/vm_api.h

```diff
@@ -5500,8 +5500,23 @@ loadWarmClassFromSnapshot(J9VMThread *vmThread, J9ClassLoader *classLoader, J9Cl
 void
 initializeSnapshotJ9Class(J9JavaVM *javaVM, J9Class *clazz);
 
+/**
+ * Perform initialization of base classes.
+ *
+ * @param javaVM[in] pointer to J9JavaVM
+ */
+void
+initializeBaseClasses(J9JavaVM *javaVM);
 #endif /* defined(J9VM_OPT_SNAPSHOTS) */
 
+/**
+ * Iterate over a class segment.
+ *
+ * @param nextClass[in] pointer to pointer of J9Class
```

Comment:
* "of" should be "to"
* the usage should be "in/out"
* the return value should be documented
```c
 * @param nextClass[in/out] pointer to pointer to J9Class
 * @return a pointer to the next class or NULL
```